sdk_contrib/typed_span/src/main/java/io/opentelemetry/sdk/contrib/typedspan/http/HttpStatusCode.java

```diff
@@ -0,0 +1,221 @@
+  /** <tt>412 Precondition Failed</tt> (HTTP/1.1 - RFC 2616) */
+  public static final int SC_PRECONDITION_FAILED = 412;
+  /** <tt>413 Request Entity Too Large</tt> (HTTP/1.1 - RFC 2616) */
+  public static final int SC_REQUEST_TOO_LONG = 413;
+  /** <tt>414 Request-URI Too Long</tt> (HTTP/1.1 - RFC 2616) */
+  public static final int SC_REQUEST_URI_TOO_LONG = 414;
+  /** <tt>415 Unsupported Media Type</tt> (HTTP/1.1 - RFC 2616) */
+  public static final int SC_UNSUPPORTED_MEDIA_TYPE = 415;
+  /** <tt>416 Requested Range Not Satisfiable</tt> (HTTP/1.1 - RFC 2616) */
+  public static final int SC_REQUESTED_RANGE_NOT_SATISFIABLE = 416;
+  /** <tt>417 Expectation Failed</tt> (HTTP/1.1 - RFC 2616) */
+  public static final int SC_EXPECTATION_FAILED = 417;
+
+  /**
+   * Static constant for a 418 error. <tt>418 Unprocessable Entity</tt> (WebDAV drafts?) or <tt>418
+   * Reauthentication Required</tt> (HTTP/1.1 drafts?)
+   */
+  // not used
+  // public static final int SC_UNPROCESSABLE_ENTITY = 418;
+
+  /**
+   * Static constant for a 419 error. <tt>419 Insufficient Space on Resource</tt> (WebDAV -
+   * draft-ietf-webdav-protocol-05?) or <tt>419 Proxy Reauthentication Required</tt> (HTTP/1.1
+   * drafts?)
+   */
+  public static final int SC_INSUFFICIENT_SPACE_ON_RESOURCE = 419;
+  /**
+   * Static constant for a 420 error. <tt>420 Method Failure</tt> (WebDAV -
+   * draft-ietf-webdav-protocol-05?)
+   */
+  public static final int SC_METHOD_FAILURE = 420;
+  /** <tt>422 Unprocessable Entity</tt> (WebDAV - RFC 2518) */
+  public static final int SC_UNPROCESSABLE_ENTITY = 422;
+  /** <tt>423 Locked</tt> (WebDAV - RFC 2518) */
+  public static final int SC_LOCKED = 423;
+  /** <tt>424 Failed Dependency</tt> (WebDAV - RFC 2518) */
+  public static final int SC_FAILED_DEPENDENCY = 424;
+
+  // --- 5xx Server Error ---
+
+  /** <tt>500 Server Error</tt> (HTTP/1.0 - RFC 1945) */
+  public static final int SC_INTERNAL_SERVER_ERROR = 500;
+  /** <tt>501 Not Implemented</tt> (HTTP/1.0 - RFC 1945) */
+  public static final int SC_NOT_IMPLEMENTED = 501;
+  /** <tt>502 Bad Gateway</tt> (HTTP/1.0 - RFC 1945) */
+  public static final int SC_BAD_GATEWAY = 502;
+  /** <tt>503 Service Unavailable</tt> (HTTP/1.0 - RFC 1945) */
+  public static final int SC_SERVICE_UNAVAILABLE = 503;
+  /** <tt>504 Gateway Timeout</tt> (HTTP/1.1 - RFC 2616) */
+  public static final int SC_GATEWAY_TIMEOUT = 504;
+  /** <tt>505 HTTP Version Not Supported</tt> (HTTP/1.1 - RFC 2616) */
+  public static final int SC_HTTP_VERSION_NOT_SUPPORTED = 505;
+
+  /** <tt>507 Insufficient Storage</tt> (WebDAV - RFC 2518) */
+  public static final int SC_INSUFFICIENT_STORAGE = 507;
+
+  private static final Map<Integer, String> code2text = new HashMap<>();
+
+  // ----------------------------------------------------- Static Initializer
+
+  /** Set up status code to "reason phrase" map. */
+  static {
+    // HTTP 1.0 Server status codes -- see RFC 1945
+    addStatusCodeMap(SC_OK, "OK");
```

Comment:
I don't think we need to add the status description. The status_text is optional and I think it should be left out if it adds no information beyond the status code integer.